### PR TITLE
Compose from source

### PR DESCRIPTION
**Pull request by sambaala**
Hello,
We want to add two new features to the R Acumos client:
1) Have a notion of R source code and be able to add it in the model bundle (cf. https://jira.acumos.org/browse/ACUMOS-3776)
2) Be able to compose a bundle from the R source (cf. https://jira.acumos.org/browse/ACUMOS-3972)

Please review these changes to check if they fulfill the need.
Regards,
Alassane Samba

### CODE REVIEW

**Comment by PhilippeDo** [comment]
Just for information, these works are part of the Demeter relase of Acumos.

More information in Jira : 

https://jira.acumos.org/browse/ACUMOS-3972
https://jira.acumos.org/browse/ACUMOS-3776

**Review by tmanoop — APPROVED** [review]

**Comment by guyjacobson** [comment]
How can I join the committers for this repo?

**Review by guyjacobson — COMMENTED** [review]
The email given for Simon Urbanek in the author section is no longer current. Should be simon.urbanek@r-project.org now.

File: DESCRIPTION [hunk 1]
```diff
@@ -1,9 +1,11 @@
 Package: acumos
 Title: Acumos R interface
-Version: 0.2-8
-Author: Simon Urbanek <urbanek@research.att.com>
```
Comment (guyjacobson) [inline DESCRIPTION:4]: This is where it should be: Simon Urbanek <simon.urbanek@r-project.org>

**Pull request merged**